tests/test_model.py

```diff
@@ -358,15 +384,13 @@ def test_available_ansaetze() -> None:
 @pytest.mark.unittest
 def test_multi_input() -> None:
     input_cases = [
-        np.random.rand(1),
         np.random.rand(1, 1),
         np.random.rand(1, 2),
         np.random.rand(1, 3),
         np.random.rand(2, 1),
         np.random.rand(20, 1),
     ]
     input_cases = [2 * np.pi * i for i in input_cases]
-    input_cases.append(None)
```

Comment:
I think, we should not remove the None input in the test. I made a suggestion to add it back in 46166c9.

Reply:
agree